pepper-apis/studybuilder-cli/studies/singular/i18n/en.conf

```diff
@@ -1619,16 +1619,16 @@
       "l4notes": """<b>*Please note</b> that this method will result in significant delays your participation in the study."""
       "p2": """Relevant medical records include operation notes, last cardiology clinic note, echo reports, and catheterization reports."""
     },
     "medical_record_release_our_staff_contact_directly": "If you are unable to provide medical records through any of the options noted above, you may choose to have the study staff reach out to your physician(s) directly. Please note that this method will result in significant delays in $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"your child's\", \"your\") participation in the study."
     "contact_for_help": "Participants with questions about this form may contact a member of the study staff at <a href='tel:650-561-6750' class='Link'>650-561-6750</a>, Monday - Friday 9 a.m. - 5 p.m. or email at <a href='mailto:contact@projectsingular.org' class='Link'>contact@projectsingular.org</a>";
-    "physician_list_intro": "Enter the physician, hospital, clinic, or provider where you would like your medical records to be sent from.",
+    "physician_list_intro": "Enter the physician, hospital, clinic, or provider where you would like $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"your child's\", \"your\") medical records to be sent from.",
     "sectionc": {
       "title": "C.",
-      "p1": "By completing this information, you are agreeing to allow us to receive $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"your child’s\", \"your\") medical records through the option you selected above (sending in $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"your child’s\", \"your\") records via email, fax, or upload; working with Project Singular study staff to help you access $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"your child’s\", \"your\") records); or having our study staff contact your designated physician(s) and hospital(s)/institutions(s) to obtain your $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"your child’s\", \"your\")).",
+      "titleText": "By completing this information, you are agreeing to allow us to receive $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"your child’s\", \"your\") medical records through the option you selected above (sending in $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"your child’s\", \"your\") records via email, fax, or upload; working with Project Singular study staff to help you access $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"your child’s\", \"your\") records); or having our study staff contact $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"your child's\", \"your\") designated physician(s) and hospital(s)/institutions(s) to obtain your $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"your child’s\", \"your\")).",
       "p2": "By signing this form, I am acknowledging that:",
```

Comment:
I had put the p1 text as, well, a separate paragraph. Request was to have with same font as title. It is done.